cmd/ci-operator/main.go

```diff
@@ -87,6 +89,7 @@ type options struct {
 
 func bindOptions() *options {
 	opt := &options{}
+	flag.Var(&opt.targets, "target", "A set of names in the config to target. Only steps that are required for these targets will be run.")
```

Comment:
What's the use-case for more than one leaf here?

Reply:
When you are building images, just run the build job to completion. When you need a leaf, you are running a single test. If the test requies multiple parents, make the tree relationship as necessary? Using `--target` for non-test runs seems wrong.